pages/user/index.html

```diff
@@ -12,8 +27,6 @@
     <script src="/node_modules/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
```

Comment:
You need to add the author and team name on here. Ref to the front end team discussion board :) 
Love that you have added comments for the next class
Great work @Menubrea

Reply:
Isn't line 22 - 24 what you're looking for, or am I misunderstanding?
```html
    <!-- Dev -->
    <meta name="author" content="@Menubrea" />
    <meta name="team" content="User" />
```

Reply:
Thats correct, It did not show up for me.. 
![image](https://user-images.githubusercontent.com/46559129/213863564-73508c93-5b05-4bfc-9030-cb0a0c5d37bb.png)
Came up when i clicked this 
![image](https://user-images.githubusercontent.com/46559129/213863574-3f6227e9-053b-4b55-9fcc-1ef4f0834e90.png)

Sorry, my mistake!

Reply:
All good. I appreciate you checking over. If there's anything I can contribute with, let me know. I feel quite invested in this project.